optimum/exporters/onnx/__main__.py

```diff
@@ -115,12 +128,20 @@ def main():
             f"At least  {onnx_config.DEFAULT_ONNX_OPSET} is required."
         )
 
-    onnx_inputs, onnx_outputs = export(
-        model,
-        onnx_config,
-        args.opset,
-        args.output,
-    )
+    use_past = True if "-with-past" in task else False
```

Comment:
I think you do not need that since it is already in the onnx config no?

Reply:
Yes, was probably thinking of how the `ORTModelForConditionalGeneration` is currently implemented which create onnx config always without past. But that can be modified later. 

Currently, updated to use `use_past` from the onnx config.